File: sem3/map/Lab/ToyLanguage/src/View/RunExample.java
package View;
import Controller.Controller;

/**
 * Created by Andrei on 09.11.2017.
 */
public class RunExample extends Command {
    private Controller ctr;
    public RunExample(String key, String desc,Controller ctr){
        super(key, desc);
        this.ctr=ctr;
    }
    @Override
    public void execute() {
        ctr.allSteps(); }
    }

File: sem3/sem 3/~craciunf/c-sharp-toy-language-interpreter/view/RunExample.cs
/* package view; */
/* import ctrl.*; */
using System.IO;
using System;

namespace ToyLanguageInterpreter {
    public class RunExample : Command {
        private Controller ctrl;

        public RunExample(String key, String description, Controller ctrl):base(key, description) {}

        /* @Override */
        public override void execute() {

            Console.WriteLine("pula");

            try {
                ctrl.allStep();
            } catch(Exception e) {
                Console.WriteLine("Errors occurred when running the program.");
                Console.WriteLine(e.StackTrace);
                Environment.Exit(1);
            }
        }
    }
}
